Makefile

```diff
@@ -64,8 +64,11 @@ container-image-single: pre-build
 release-snapshot:
 	@./hack/goreleaser.sh release_snapshot
 
+build-single-arch-cli:
+	@./hack/goreleaser.sh build_single_arch_cli
+
 release-snapshot-cli:
```

Comment:
It needs to stay because of the `pull_request_target` that always uses the job manifest from `main`. I will remove it in my next PR after merging this one.